src/org/isf/patient/service/PatientIoOperationRepositoryImpl.java

```diff
@@ -0,0 +1,113 @@
+package org.isf.patient.service;
```

Comment:
Missing license header.

Reply:
This review is not resolved, @ArnaudFonzam you need to add the License header to all new files

Reply:
ok, the whole file has been deleted